srcpkgs/alass/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'alass'
+pkgname=alass
+version=2.0.0
+revision=1
+build_style=cargo
+depends="ffmpeg"
+short_desc="Command line tool to synchronize subtitles to movies"
+maintainer="Dakota Richline <drichline@protonmail.com>"
+license="GPL-3.0-only"
+homepage="https://github.com/kaegi/alass"
+distfiles="https://github.com/kaegi/alass/archive/refs/tags/v${version}.tar.gz"
+checksum=ce88f92c7a427b623edcabb1b64e80be70cca2777f3da4b96702820a6cdf1e26
+
+do_install() {
+	vbin target/${RUST_TARGET}/release/alass-cli
+	vdoc README.md
+}
+
+if [ "$XBPS_TARGET_MACHINE" = "i686" ]; then
+	make_check=ci-skip # Tests fail to build on i686 container with error "ran out of registers during register allocation"
+fi
```

Comment:
let's use a case statement here instead because, while we don't currently have i686-musl, we should still consider it

and also change the `ci-skip` to `no` because it fails locally too

```suggestion
case "$XBPS_TARGET_MACHINE" in
	i686*)
		make_check=no # Tests fail to build on i686 with error "ran out of registers during register allocation"
		;;
esac
```

and, as a stylistic thing, let's put this before the functions and after the main variable block